cmd/crc/cmd/start.go

```diff
@@ -191,20 +194,39 @@ func validateStartFlags() error {
 	return nil
 }
 
-func checkIfNewVersionAvailable(noUpdateCheck bool) {
+func checkIfNewVersionAvailable(noUpdateCheck bool) error {
 	if noUpdateCheck {
-		return
+		return nil
 	}
-	isNewVersionAvailable, newVersion, err := crcversion.NewVersionAvailable()
+	isNewVersionAvailable, newVersion, link, err := newVersionAvailable()
 	if err != nil {
-		logging.Debugf("Unable to find out if a new version is available: %v", err)
-		return
+		return err
 	}
 	if isNewVersionAvailable {
-		logging.Warnf("A new version (%s) has been published on %s", newVersion, constants.CrcLandingPageURL)
-		return
+		logging.Warnf("A new version (%s) has been published on %s", newVersion, link)
+		return nil
 	}
 	logging.Debugf("No new version available. The latest version is %s", newVersion)
+	return nil
+}
+
+func newVersionAvailable() (bool, string, string, error) {
```

Comment:
A short comment describing the return values might be useful as `(bool, string, string, error)` is a bit complex